test/core/derivatives/rnn_settings.py

```diff
@@ -17,11 +17,15 @@
 
 RNN_SETTINGS = [
     {
-        "module_fn": lambda: torch.nn.RNN(input_size=4, hidden_size=2),
-        "input_fn": lambda: torch.rand(size=(5, 7, 4)),
+        "module_fn": lambda: torch.nn.RNN(
+            input_size=4, hidden_size=2, batch_first=True
+        ),
+        "input_fn": lambda: torch.rand(size=(7, 5, 4)),
     },
     {
-        "module_fn": lambda: torch.nn.RNN(input_size=4, hidden_size=2),
+        "module_fn": lambda: torch.nn.RNN(
+            input_size=4, hidden_size=2, batch_first=True
+        ),
```

Comment:
Would be ok to delete these cases because the one below is similar except for dimensions.